### PR TITLE
*.py: python codes format

### PR DESCRIPTION
**Pull request by jnozsc**
Read this before you submit this pull request:
https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md

Reading this link and following the rules will get your pull request reviewed
and merged faster. Nobody wants lazy pull requests.

### CODE REVIEW

File: TOOLS/osxbundle.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-
+from __future__ import print_function
```
Comment (CounterPillow) [inline TOOLS/osxbundle.py]: python 2 is dead anyway though, are you sure this is needed and the additional space between `print` and `(` wasn't just a typo?

Comment (jnozsc) [reply TOOLS/osxbundle.py]: this is not necessary for the change on line 40, but I saw this  `from __future__ import ` from other files, so I assume a py2 compatibility statement is required for this repo.   I can remove this line.

Comment (jnozsc) [reply TOOLS/osxbundle.py]: removed in https://github.com/mpv-player/mpv/pull/7354/commits/9ad9f1d6d3365f6d5f76d4da9ed841075a13ecfd

**Comment by jnozsc** [comment]
amended

**Comment by wm4** [comment]
So can anyone bring this into a merge ready form? Or I'll have to close this.

**Comment by jnozsc** [comment]
anything I can do for this PR? I think I have already amend the comment message format, lmk anything else I can do

**Comment by wm4** [comment]
These are 3 commits, but they all seem to do both of these:
- modernizing the code to python 3 conventions by dropping python 2 compatibility
- cosmetic changes, especially whitespace changes

These changes should probably be reorganized into 2 commits that does only one of each (shouldn't be hard; use `git add -p` or better some GUI thing that lets you select which lines to add to a commit, I prefer using git-cola for this purpose).

Also, I have severe doubts whether dropping python2 compatibility in waf's current state is a good idea. The current version of waf that we use has `env python` in its shebang, which means on most distros (except ArchLinux), it will invoke python 2, and load the waftools modules as python 2 as well. This would be ok only if waf is going to explicitly use python 3 (maybe a newer release does?).

**Comment by jnozsc** [comment]
I add more `drop python2 compatibility ` since CounterPillow mentioned above as `python 2 is dead anyway though, are you sure this is needed `. I thought I should add it to this PR. 

If we are not agree with this change, I could only have one PR to clean the python code format only.

**Comment by wm4** [comment]
That would probably be better for now. The commit message should also be reworded.

**Comment by jnozsc** [comment]
ok, close this pr for now. thanks

**Comment by jnozsc** [comment]
new pr https://github.com/mpv-player/mpv/pull/7495

**Pull request closed without merging**